### DISCUSSION TITLE
Which of the UniRef50 clusers were held out for evalutation set?

### DISCUSSION THREAD

**Discussion by spetti** [facebookresearch/esm#39] [General] [answered]
The paper says that 10% of the UniRef50 clusters were randomly selected as a held-out evaluation set. Which clusters are in this 10%? Thanks!

**Comment by tomsercu** [comment, marked as answer]
Hi spetti,
thanks for your interest! We just added and documented the pre-training splits [here in the README](https://github.com/facebookresearch/esm#pre-training-dataset-split--).
Hope this is helpful.
Tom
